blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -0,0 +1,12 @@
+bo.dashboard.pipelines.heading=Dashboard
+bo.dashboard.pipelines.new.pipeline=New Pipeline
+bo.dashboard.pipelines.table.health=Health
+bo.dashboard.pipelines.table.branches=Branches
+bo.dashboard.pipelines.table.pr=Pull Requests
+bo.dashboard.loading=Loading...
+bo.dashboard.more=Show more
+bo.dashboard.failing={0} failing
```

Comment:
see my comments in this PR about the named parameters. in java properties files you only can use numeric values. 

However since i18next is very flexible, you can do ```##name##``` and then in your client side code

```
    t('who.rocks', {
        name: 'Tom the man',
        interpolation: {
            prefix: '##',
            suffix: '##',
        }
    })
}
```

....and what do you mean with server side? Current state of the art in jenkins are those properties file - no ```{name}``` in any of those for sure.